server/channels/store/sqlstore/integrity.go

```diff
@@ -544,3 +547,61 @@ func CheckRelationalIntegrity(ss *SqlStore, results chan<- model.IntegrityCheckR
 	mlog.Info("Done with relational integrity checks")
 	close(results)
 }
+
+func GetAllValidUserIDsFromDMChannels(ss *SqlStore) ([]*model.User, error) {
+	records := []model.OrphanedRecord{}
+	dirtyIDs := []string{}
+	query, args, err := ss.getQueryBuilder().
+		Select().
+		Column("CT.ChannelName AS ParentId").
+		From("Channels AS CT").
+		Where(sq.Eq{"CT.Type": []model.ChannelType{model.ChannelTypeDirect}}).
+		ToSql()
+
+	if err != nil {
+		mlog.Error("There is an error with the DM Channel Name query")
+		return nil, err
+	}
+
+	err = ss.GetMasterX().Select(&records, query, args...)
```

Comment:
You can directly use `SelectBuilder` and pass a builder to it. No need to do `ToSQL` and check error.